Antlr3.vs2015.sln

```diff
@@ -0,0 +1,238 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
:thought_balloon: I would prefer to not add a new solution file, since it would confuse new contributors regarding which one they should be working in.